controllers/component_image_controller.go

```diff
@@ -295,6 +257,41 @@ func (r *ComponentImageReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{}, nil
 }
 
+func getImageReferenceFromBuildPipeline(buildPipelineRun tektonapi.PipelineRun) (string, error) {
+	pipelineResults := buildPipelineRun.Status.PipelineResults
```

Comment:
ok cool you sorted out your "how to get pipeline results from status" question you had yesterday :-)